### DISCUSSION TITLE
Maximum hotend temperature

### DISCUSSION THREAD

**Discussion by armadi1969** [Jyers/Marlin#1936] [Ideas]
Hello I would like to ask, sorry i'm new and i'm trying to learn but i'm asking you ... all these modifications of all metal hotends and more ... it is not possible to have the possibility to increase the maximum temperature ... for the minimum there is .. I do not see why it cannot be done ... like 280°

**Comment by Cal-LF** [comment]
You can search the issues and discussions areas. There are several posts concerning modifying max temp. Look for "max temp" "hi temp" ect.

**Reply by armadi1969** [reply]
thanks
